bigchaindb/web/views/statuses.py

```diff
@@ -0,0 +1,62 @@
+"""This module provides the blueprint for the statuses API endpoints.
+
+For more information please refer to the documentation on ReadTheDocs:
+ - https://docs.bigchaindb.com/projects/server/en/latest/drivers-clients/
+   http-client-server-api.html
+"""
+from flask import current_app
+from flask_restful import Resource, reqparse
+
+from bigchaindb.web.views.base import make_error
+
+
+class StatusApi(Resource):
+    def get(self):
+        """API endpoint to get details about the status of a transaction or a block.
+
+        Return:
+            A ``dict`` in the format ``{'status': <status>}``, where
+            ``<status>`` is one of "valid", "invalid", "undecided", "backlog".
+        """
+        parser = reqparse.RequestParser()
+        parser.add_argument('tx_id', type=str)
+        parser.add_argument('block_id', type=str)
+
+        args = parser.parse_args(strict=True)
+
+        if sum(arg is not None for arg in args.values()) != 1:
```

Comment:
I think what you're looking for is an exclusive OR, right?

I don't have any experience with that in Python but perhaps you can do smth like:

- http://stackoverflow.com/a/433161/1263876
- http://stackoverflow.com/a/432844/1263876

Alternatively you can express an xor with the built in python operators.